### ISSUE TITLE
Failure to install dependencies when not installing latest in the repo

### ISSUE THREAD

**Issue by ghost** [Icinga/chef-icinga2client#6]
We set the version attribute in a recipe `node.default['icinga2client']['version'] = '2.9.2-1'` and install via `include_recipe 'icinga2client::install`.

Apt does not install the dependencies at the same version so we should install these packages together in one `apt-get install` run.

```bash
root@defaultubuntu18-nologin-zmvm-pqhoh07:~# apt-get install icinga2=2.9.2-1.bionic
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 icinga2 : Depends: icinga2-bin (= 2.9.2-1.bionic) but 2.10.0-1.bionic is to be installed
           Depends: icinga2-common (= 2.9.2-1.bionic) but 2.10.0-1.bionic is to be installed
           Recommends: icinga2-doc (= 2.9.2-1.bionic) but it is not going to be installed
           Recommends: monitoring-plugins-basic but it is not going to be installed or
                       nagios-plugins-basic but it is not going to be installed
           Recommends: libreadline6 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

I suggest something like...
```ruby
# Setup the array of packages we want to install in one apt-get install run.
packages = %w[
  icinga2
  icinga2-bin
  icinga2-common
  icinga2-doc
]

versions = []
# The package resource, when installing from an array of packages, requires a version
# per package in the array even if the versions are the same. So we set up the array of versions here.
packages.each do |_element|
  versions << "#{node['icinga2client']['version']}#{node['icinga2client']['version_suffix']}"
end

# Install icinga2, icinga2-bin, icinga2-common, icinga2-doc packages at 
# version: "#{node['icinga2client']['version']}#{node['icinga2client']['version_suffix']}"
package packages do
  version versions unless node['icinga2client']['ignore_version']
  notifies :restart, 'service[icinga2]', :delayed
end
```

**Comment by ghost** [comment]
I just noticed there's a PR for this already:
https://github.com/Icinga/chef-icinga2client/pull/5/

**Comment by ghost** [comment]
I'm going to close this PR. #5 should fix this too @vkhatri

**Comment by vkhatri** [comment]
@zmaupin-do I have just merged the PR #5 

Thank you for the PR though.
